common/src/main/java/dev/ftb/mods/ftbchunks/client/config/StringMapValue.java

```diff
@@ -0,0 +1,40 @@
+package dev.ftb.mods.ftbchunks.client.config;
```

Comment:
This feels like it should be in FTB Library (even if only used by chunks for now)